packages/eslint-config-wpcalypso/.eslintrc.js

```diff
@@ -0,0 +1,8 @@
+/** @format */
+
+module.exports = {
+	extends: '../../.eslintrc.js',
```

Comment:
These can actually be removed, [eslint will look in parent directories until a the root dir or a `root: true` config.](https://eslint.org/docs/user-guide/configuring#using-configuration-files-1)